pkg/ccl/backupccl/testdata/backup-restore/temp-tables

```diff
@@ -146,14 +145,12 @@ pg_catalog
 pg_extension
 public
 
-# On full cluster restore we restore temp tables to its original database.
+# On full cluster restore we do not restore temp tables
 query-sql
 USE d1;
 SELECT table_name FROM [SHOW TABLES] ORDER BY table_name
 ----
 perm_table
-temp_seq
-temp_table
 
 # Nudge and wait for the temp cleanup job to run.
```

Comment:
It seemed like it was showing that we didn't have a tmp table post-restore, so that was worth keeping, but I could be convinced it isn't. It can also go away later?

Reply:
Done.